packages/flutter/lib/src/material/drawer.dart

```diff
@@ -217,7 +221,18 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
   }
 
   void _move(DragUpdateDetails details) {
-    final double delta = details.primaryDelta / _width;
+    if (_controller.isDismissed) {
+      final ModalRoute<dynamic> parentRoute = ModalRoute.of(context);
+      final bool canPop = parentRoute?.canPop ?? false;
+      if(canPop) {
+        Navigator.of(context).pop();
+        return;
+      }
+    }
+    double delta = details.primaryDelta / _width;
+    if(widget.type == DrawerAlignment.END) {
+      delta *= -1;
```

Comment:
Please use unary negation rather than `*= -1`.  It's both faster and semantically clearer.

Reply:
Fixed